spec/requests/admin/email_templates_controller_spec.rb

```diff
@@ -29,6 +34,12 @@ def original_text(key)
       expect(response.status).to eq(404)
     end
 
+    it "raises an error if you are a moderator" do
```

Comment:
```suggestion
    it "returns 404 when user is a moderator" do
```

Two suggestions here. The first is that returning 404 isn't really an error but a valid HTTP response code. The second is that we usually write in the third person perspective instead of the second.